src/common/services/user/index.ts

```diff
@@ -32,8 +32,8 @@ type LoginResponse = {
 };
 
 type UserState = {
-  profile?: Profile;
-  token?: string;
+  profile: Profile | null;
```

Comment:
why null instead of optional?

Reply:
cekni redux docs

Reply:
mam ich cele precitane, neviem na co narazas

Reply:
skus aby reducer vracal undefined a uvidis